client/trino-cli/src/main/java/io/trino/cli/JsonPrinter.java

```diff
@@ -70,4 +72,17 @@ private static Object formatValue(Object o)
         }
         return o;
     }
+
+    @SuppressModernizer
```

Comment:
Please provide a comment why it is suppressed

Reply:
That's overly broad. Does it suppress *all* modernizer checks?

Reply:
Yes. There is no way to disable a single rule